ntoskrnl/mm/ARM3/expool.c

```diff
@@ -62,7 +62,7 @@ ULONGLONG MiLastPoolDumpTime;
  * pool code, but only for checked builds.
  *
  * As of Vista, however, an MSDN Blog entry by a Security Team Manager indicates
- * that these checks are done even on retail builds, due to the increasing
+ * that these checks are done even on retail builds, dNonPagedPoolDescriptorue to the increasing
```

Comment:
There appears to be an obvious copy-paste error here in the middle of 'due'.

Reply:
Fixed, thanks !